nix/cells/plutus/packages/plutus-metatheory-site.nix

```diff
@@ -0,0 +1,35 @@
+{ inputs, cell }:
+
+let
+  # Doing this in two derivations so the call to agda is cached, since
+  # that's very slow. Makes it easier to iterate on the site build.
+  plutus-metatheory-agda-html = cell.library.pkgs.stdenv.mkDerivation {
+    name = "plutus-metatheory-doc";
+    # Not doing this seems to lead to spurious rebuilds, I think because it
```

Comment:
FYI @zeme-iohk this is the thing I slacked you about. Seems suspicious.

Reply:
That's odd, we use the pattern `inputs.self + /sub/dir` in many other places. 🤔